docs/tuning.md

```diff
@@ -264,6 +264,13 @@ parent RDD's number of partitions. You can pass the level of parallelism as a se
 or set the config property `spark.default.parallelism` to change the default.
 In general, we recommend 2-3 tasks per CPU core in your cluster.
 
+Sometimes you may also need to increase directory listing parallelism when job input has large number of directories,
+otherwise the process could take a very long time, especially when against object store like S3.
+If your job works on RDD with Hadoop input formats (e.g., via `SparkContext#sequenceFile`), the parallelism is
```

Comment:
nit: `SparkContext#sequenceFile` -> `SparkContext.sequenceFile` to be consistent in this page.

Reply:
+1

Reply:
+1

Reply:
👍